website/docs/add-secure-apps/flows-stages/stages/mtls/index.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Mutual TLS stage
+authentik_version: "2025.6"
+authentik_preview: true
+authentik_enterprise: true
+toc_max_heading_level: 5
+---
+
+The Mutual TLS stage enables authentik to authenticate or enroll users utilizing client certificates. Management of client certificates is out of the scope of this document.
+
+## Reverse-proxy configuration
+
+Using this stage requires special configuration of any reverse proxy that is used in front of authentik, as the reverse-proxy interacts directly with the browser.
+
+- nginx
+    - [Standalone nginx](#nginx-standalone)
+    - [nginx kubernetes ingress](#nginx-ingress)
+- Traefik
+    - [Standalone Traefik](#traefik-standalone)
+    - [Traefik kubernetes ingress](#traefik-ingress)
+- [envoy](#envoy)
+- [No reverse proxy](#no-reverse-proxy)
+
+#### nginx Standalone
+
+Add this configuration snippet in your authentik virtual host:
+
+```nginx
+# server {
+    ssl_client_certificate /etc/ssl/path-to-my-ca.pem;
+    ssl_verify_client on;
+
+    # location / {
+        proxy_set_header ssl-client-cert $ssl_client_escaped_cert;
+    # }
+# }
+```
+
+See [nginx documentation](https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_client_certificate) for reference.
+
+#### nginx Ingress
+
+Add these annotations to your authentik ingress:
```

Comment:
... to your authentik ingress definition/configuration/file/..?? We need a noun after "ingress". ;-)

Reply:
to follow the other k8s example below, probably `object`